src/Compilers/CSharp/Portable/Binder/Binder_Attributes.cs

```diff
@@ -330,6 +330,7 @@ protected virtual bool IsAttributeConditionallyOmitted(NamedTypeSymbol attribute
                         }
 
                         BoundExpression boundNamedArgument = BindNamedAttributeArgument(argument, attributeType, diagnostics);
+                        this.Compilation.VariableUsePass.Visit(boundNamedArgument);
```

Comment:
Marking this call to fail if it's a local function, then running all unit tests, says that no, this code is not getting hit on attributes on local functions, including parameters and type parameters.